lib/Db/Item.php

```diff
@@ -199,7 +199,7 @@ public function getEnclosureMime(): ?string
         return $this->enclosureMime;
     }
 
-    public function getFeedId(): string
+    public function getFeedId(): int
```

Comment:
Seems right, but does this affect the generated API responses?

Reply:
If it does I'd rather cast it again in the controller for v1.